### PR TITLE
Bug 1773665: Backport changes from BZ 1760393 to 4.2.z

### PR DESCRIPTION
**Pull request by chambridge**
Backport the metric whitelisting from https://github.com/openshift/telemeter/pull/255

### CODE REVIEW

File: jsonnet/telemeter/metrics.jsonnet [hunk 1]
```diff
@@ -88,9 +88,11 @@
   '{__name__="cluster:node_instance_type_count:sum"}',
   // cnv:vmi_status_running:count is the total number of VM instances running in the cluster.
   '{__name__="cnv:vmi_status_running:count"}',
-  // node_role_os_version_machine:cpu_capacity_cores:sum is the total number of CPU cores
```
Comment (squat) [inline jsonnet/telemeter/metrics.jsonnet]: I think you deleted an extra line here

Comment (chambridge) [reply jsonnet/telemeter/metrics.jsonnet]: yep. 🤦‍♂ 
Will fix.

Comment (chambridge) [reply jsonnet/telemeter/metrics.jsonnet]: Done

**Pull request merged**